css/templates/travel/components/_footer.css

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ .travel-footer {
+  background-color: var(--white);
+}
+
+.travel-footer amp-img {
+  background-color: var(--silver);
+}
+
+.travel-footer .link.gray:focus {
+  border-bottom: calc(2 / 16rem) solid rgba(255,255,255,0.25);
+}
+
+.travel-footer-angle-block {
+  left: 50%;
+}
+
+.travel-footer-right-column .blue {
+  color: var(--blue);
+}
+
+.travel-footer-input-bg {
+  position: absolute;
```

Comment:
can some of these properties be move to HTML

Reply:
please check for this all over, also let me know if there is an actual reason for retaining this here

Reply:
done